libcxx/modules/std/atomic.inc

```diff
@@ -10,132 +10,132 @@
 export namespace std {
 
   // [atomics.order], order and consistency
-  using std::memory_order;
-  using std::memory_order_acq_rel;
-  using std::memory_order_acquire;
-  using std::memory_order_consume;
-  using std::memory_order_relaxed;
-  using std::memory_order_release;
-  using std::memory_order_seq_cst;
+  using std::memory_order _LIBCPP_USING_IF_EXISTS;
```

Comment:
We provide (almost?) all of the symbols listed here unconditionally. Why are they marked `_LIBCPP_USING_IF_EXISTS`? These are also not marked in the `std.compat` module.

Reply:
I have no idea why we mark those with `_LIBCPP_USING_IF_EXISTS` in the headers. We shouldn't. However I suggest we blindly apply this patch to match the headers, and then fix the remaining issues in both the headers and the module file.

Reply:
These values are from C headers too.
I indeed added what we did for the C-headers. Before I did not add all of these and that lead to several issues. So I agree with @ldionne's approach.